workbench-core/example/infrastructure/integration-tests/tests/authentication/routeHandler.test.ts

```diff
@@ -69,4 +69,39 @@ describe('authentication route handler integration tests', () => {
       ).rejects.toThrow(new HttpError(400, {}));
     });
   });
+
+  describe('isUserLoggedIn', () => {
+    it('should return false for an invalid refresh token', async () => {
```

Comment:
nit: can be shorten with a [test.each](https://jestjs.io/docs/api#testeachtablename-fn-timeout)